src/ralph/assets/tests/factories.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
BTW to discuss if we want to reuse these factories in demo data generation process. If yes, then we should probably give them meaningful names etc. @ar4s @xliiv ?